EIPS/eip-4361.md

```diff
@@ -0,0 +1,313 @@
+---
+eip: 4361
+title: Sign-In with Ethereum.
+description: Standardizes off-chain authentication for Ethereum accounts to establish sessions.
+author: Wayne Chang (@wyc), Gregory Rocco (@obstropolos), Brantly Millegan (@brantlymillegan), Nick Johnson (@Arachnid)
+discussions-to: https://ethereum-magicians.org/t/eip-4361-sign-in-with-ethereum/7263
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-10-11
+requires: 55, 137, 155, 191, 1271, 1328
+---
+
+## Abstract
+Sign-In with Ethereum describes how Ethereum accounts authenticate with off-chain services by signing a standard message format parameterized by scope, session details, and security mechanisms (e.g., a nonce). The goals of this specification are to provide a self-custodied alternative to centralized identity providers, improve interoperability across off-chain services for Ethereum-based authentication, and provide wallet vendors a consistent machine-readable message format to achieve improved user experiences and consent management.
+
+## Motivation
+When signing in to popular non-blockchain services today, users will typically use identity providers (IdPs) that are centralized entities with ultimate control over users' identifiers, for example, large internet companies and email providers. Incentives are often misaligned between these parties. Sign-In with Ethereum offers a new self-custodial option for users who wish to assume more control and responsibility over their own digital identity.
+
+Already, many services support workflows to authenticate Ethereum accounts using message signing, such as to establish a cookie-based web session which can manage privileged metadata about the authenticating address. This is an opportunity to standardize the sign-in workflow and improve interoperability across existing services, while also providing wallet vendors a reliable method to identify signing requests as Sign-In with Ethereum requests for improved UX.
+
+This work is sponsored by the Ethereum Foundation and Ethereum Name Service (ENS). It is being developed in the open through a series of recorded community calls and public repositories, and its development is informed by over twenty user interviews with a focus on currently-in-production uses, related prior EIPs, and fits within product roadmaps.
```

Comment:
Usually we don't include this kind of information in an EIP. I recommend removing it and instead posting it in the `discussions-to` thread.

Reply:
Addressed in https://github.com/ethereum/EIPs/pull/4361/commits/e59daacf95ecf9747b47921f7ce7b865eb139f75